org.eclipse.jdt.core/dom/org/eclipse/jdt/core/dom/AST.java

```diff
@@ -534,7 +534,7 @@ public final class AST {
 	 * Internal property for latest supported JLS level
 	 * This provides the latest JLS level.
 	 */
-	private static final int JLS_INTERNAL_Latest = JLS22;
+	private static final int JLS_INTERNAL_Latest = JLS23;
```

Comment:
Should we wait for that to change in JDT?

Reply:
I do this change as without it there are test failures like https://ci.eclipse.org/ls/job/jdt-core-incubator/job/dom-with-javac/140/testReport/junit/org.eclipse.jdt.core.tests.dom/ASTTest/Build_and_Test___testASTLevels___JLS2/

Reply:
This fixed the mentioned tests but caused others to fail thus removed from this PR.